### PR TITLE
PLAT-207: add /wallet/add_amount, /wallet/get_balance tests

### PR DESCRIPTION
**Pull request by ElSacramento**
add tests to check response fields, add tests to check concurrent requests

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/application/functest/api_test_utils.go [hunk 1]
```diff
@@ -28,8 +28,10 @@ const (
 	requestTimeout = 30 * time.Second
 	contentType    = "Content-Type"
 
-	defaultHost      = "127.0.0.1"
-	createWalletPath = "/wallet/create"
+	defaultHost          = "127.0.0.1"
+	walletCreatePath     = "/wallet/create"
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/api_test_utils.go:32]: I would move "/wallet" to separate variable and reuse it. But it's not necessary.

File: ledger-core/v2/application/functest/wallet_add_amount_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	require.NoError(t, err, "failed to create wallet")
+
+	addAmountURL := getURL(walletAddAmountPath, "", "")
+	rawResp, err := sendAPIRequest(addAmountURL, walletAddAmountRequestBody{To: walletRef, Amount: 100})
+	require.NoError(t, err, "failed to send request or get response body")
+
+	resp, err := unmarshalWalletAddAmountResponse(rawResp)
+	require.NoError(t, err, "failed to unmarshal response")
+
+	assert.Empty(t, resp.Err, "problem during execute request")
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/wallet_add_amount_test.go:30]: Why do you use "assert" here? I mean why not "require"?

File: ledger-core/v2/application/functest/wallet_add_amount_test.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	walletRef, err := createSimpleWallet()
+	require.NoError(t, err, "failed to create wallet")
+
+	// Data doesn't transfer from node to node, so all requests will be send to one node.
+	var wg sync.WaitGroup
+
+	count := 10 // Number of concurrent requests per node.
+	for i := 0; i < count; i++ {
+		for _, port := range nodesPorts {
+			wg.Add(1)
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/wallet_add_amount_test.go:45]: You can do it once, before cycle

File: ledger-core/v2/application/functest/wallet_add_amount_test.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+			go func(wg *sync.WaitGroup, port string) {
+				defer wg.Done()
+
+				addAmountURL := getURL(walletAddAmountPath, "", port)
+				err := addAmountToWallet(addAmountURL, walletRef, 100)
+				assert.NoError(t, err)
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/wallet_add_amount_test.go:51]: Why not "require"?

File: ledger-core/v2/application/functest/wallet_create_test.go [hunk 5]
```diff
@@ -15,14 +15,14 @@ import (
 )
 
 // Creates wallets and check Reference format in response body.
-func TestCreateWallet(t *testing.T) {
+func TestWalletCreate(t *testing.T) {
 	status := getStatus(t)
 	require.NotEqual(t, 0, status.WorkingListSize, "not enough nodes to run test")
 	count := 50 * status.WorkingListSize // 250 for 5 nodes
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/wallet_create_test.go:21]: What information does this comment give me?

Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_create_test.go:21]: how many requests to /wallet/create will be

Comment (ivanshibitov) [reply ledger-core/v2/application/functest/wallet_create_test.go:21]: why do I need it?=) as a code reader?

File: ledger-core/v2/application/functest/wallet_add_amount_test.go (hunk 2 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_add_amount_test.go:30]: To check both fields of response and not fail after a check of one field.

File: ledger-core/v2/application/functest/wallet_add_amount_test.go (hunk 4 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_add_amount_test.go:51]: To not fail cycle due to error in one request.

File: ledger-core/v2/application/functest/wallet_add_amount_test.go (hunk 3 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_add_amount_test.go:45]: with `wg.Add(len(nodesPorts))` ?

Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_add_amount_test.go:45]: I think it's more readable to use `wg.Add(1)` right before the creation of goroutine.

File: ledger-core/v2/application/functest/api_test_utils.go (hunk 1 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/api_test_utils.go]: done

File: ledger-core/v2/application/functest/wallet_add_amount_test.go (hunk 2 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_add_amount_test.go]: changed to require

File: ledger-core/v2/application/functest/wallet_create_test.go (hunk 5 above)
Comment (ElSacramento) [reply ledger-core/v2/application/functest/wallet_create_test.go]: removed

**Pull request merged**